elastic/kibana
# [Security Solution][RAC] - Remove timestamp from reason field

## Summary

This PR removes the timestamp from the reason field as it's been considered repetitive given the timestamp column

To Test:
 - Create a rule
 - Check the alert table for the reason field

![image](https://user-images.githubusercontent.com/17211684/130465385-7f59fe0f-1d01-4911-aec8-fa159cc1776b.png)



### Checklist

Delete any items that are not applicable to this PR.
